src/FSharpVSPowerTools/FSharpVSPowerTools.csproj

```diff
@@ -78,6 +78,7 @@
     <Reference Include="Microsoft.VisualStudio.Editor, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
       <Private>False</Private>
     </Reference>
+    <Reference Include="Microsoft.VisualStudio.Language.StandardClassification, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
```

Comment:
Could you put a copy of this assembly to https://github.com/fsprojects/VisualFSharpPowerTools/tree/master/lib/vs2013? It will work on machines without VS SDK 2013 installed (e.g. those with VS SDK 2015 only).